src/pages/usage.md

```diff
@@ -0,0 +1,78 @@
+* [Google Refine](https://scholar.google.ca/scholar?hl=en&as_sdt=0%2C5&as_ylo=2023&as_yhi=2023&q=%22Google+Refine%22+-openrefine&btnG=)
+* [Open Refine](https://scholar.google.ca/scholar?hl=en&as_sdt=0%2C5&as_ylo=2023&as_yhi=2023&q=%22Open+Refine%22+-openrefine&btnG=)
+* [OpenRefine](https://scholar.google.ca/scholar?hl=en&as_sdt=0%2C5&as_ylo=2023&as_yhi=2023&q=%22OpenRefine%22&btnG=)
+
+
+```mermaid 
+%%{init: {'theme':'forest'}}%%
+xychart-beta
+    title "Number of academic citations of OpenRefine per years"
+    x-axis [2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023]
+    y-axis "Count of citation" 0 --> 900
+    bar [4, 67, 162, 251, 356, 424, 477, 502, 584, 571, 586, 717, 783, 811]
+```
+
+## Forum Statistics 
+
+On November 2022, we moved from our Google Groups to discourse to host our forum. 
+
+As of March 20, 2024, over the last 12 months
+* 244 new users signed up on our forum for a total of 470 users. 
+* 324 topics were created for a total of 1,700 messages.
+
+## Contribution Statistics
+
+The statistics below only track activity on our primary GitHub repository, which includes code, design, and translation contributions. Our documentation is available in a separate repository.
+
+As of March 20th, 2024, over the last 12 months we had: 
+* 41 active GitHub contributors;
+* 227 issues [created](https://github.com/OpenRefine/OpenRefine/issues?q=is%3Aissue+created%3A2023-03-20..2024-03-20) and 199 [closed](https://github.com/OpenRefine/OpenRefine/issues?q=is%3Aissue+closed%3A2023-03-20..2024-03-20);
+* [223 PRs merged](https://github.com/OpenRefine/OpenRefine/pulls?page=3&q=is%3Amerged+created%3A2023-03-20..2024-03-20+-author%3Aapp%2Fdependabot) (excluding those created by dependabot) 
+
+The following graphic represent a three months rolling average of the number of active contributors on the main repository [^1]. You can also review the [Github pulse](https://github.com/OpenRefine/OpenRefine/pulse) and [Github traffic](https://github.com/OpenRefine/OpenRefine/graphs/traffic) pages for real time insights on the activities on our main repository. 
+
+```mermaid 
+%%{init: {'theme':'forest'}}%%
+xychart-beta 
+    x-axis   2010 --> 2023
+    y-axis "Number of GitHub contributors per month" 0 --> 30 
+    line [1,2,2,3,4,4,3,2,3,4,4,4,3,3,2,2,2,2,2,2,2,3,3,3,2,2,2,1,1,1,1,2,2,2,2,3,3,2,1,2,2,3,2,2,1,1,1,2,2,2,2,4,4,3,4,4,3,3,2,4,4,5,3,2,0,0,0,0,1,1,1,1,1,2,1,1,2,3,4,4,3,5,6,6,8,8,11,11,14,14,15,14,11,9,7,8,9,12,13,12,8,7,7,7,5,6,6,8,8,9,7,8,9,17,21,22,21,19,20,19,17,18,14,15,16,19,20,17,17,12,9,9,12,15,15,16,16,18,19,17,15,14,15,12,11,12,13,11,10,11,12,13,12,10,8,8,8,9,8,9]
+```
+
+
+[^1]: We are counting the number of contributor using the following command `git log --all --pretty="%an" | sort | uniq | wc -l`
```

Comment:
I'm not sure how easy this would be to filter, but a cursory check shows that it includes a number of bots (dependabot x2, weblate x4, etc) and dupes (Abbe98/Albin Larsson, Jacky/jackyq2015/Qi Cui, kush x2, etc) .

This is fine for this pass however.

Reply:
I agree. I took note of this when we refresh the statistic